README.md

```diff
@@ -10,21 +10,338 @@ JavaScript XHR
+      "avatar_url": "https://avatars.githubusercontent.com/u/583231?v=3",
+      "gravatar_id": "",
+      "url": "https://api.github.com/users/octocat",
+      "html_url": "https://github.com/octocat",
+      "followers_url": "https://api.github.com/users/octocat/followers",
+      "following_url": "https://api.github.com/users/octocat/following{/other_user}",
+      "gists_url": "https://api.github.com/users/octocat/gists{/gist_id}",
+      "starred_url": "https://api.github.com/users/octocat/starred{/owner}{/repo}",
+      "subscriptions_url": "https://api.github.com/users/octocat/subscriptions",
+      "organizations_url": "https://api.github.com/users/octocat/orgs",
+      "repos_url": "https://api.github.com/users/octocat/repos",
+      "events_url": "https://api.github.com/users/octocat/events{/privacy}",
+      "received_events_url": "https://api.github.com/users/octocat/received_events",
+      "type": "User",
+      "site_admin": false
+    },
+//... more!
+```
+
+It worked! We successfully fetched data from a remote resource with XHR without
+reloading our page!
+
+Now that we have the request part down, we need to figure out how to
+capture this response so we can do something with it.
+
+### Handling the XHR Response
+
+The second part of XHR is handling the response once we've made the
+request. We do this by defining an event listener on the request to
+listen for the `load` event, which will tell us that the request is
+complete. We'll give this listener a *callback function*, which is
+simply a function that will get called when the event fires.
+
+```js
+function showRepositories(event, data) {
+  //this is set to the XMLHttpRequest object that fired the event
+  console.log(this.responseText)
+}
+
+function getRepositories() {
+  var req = new XMLHttpRequest()
+  req.addEventListener("load", showRepositories);
+  req.open("GET", 'https://api.github.com/users/octocat/repos')
+  req.send()
+}
+```
+
+When we add the event listener to our `req` object, we set it up so that
+`this` will be our `req` object inside our callback function. So, inside
+`showRepositories`, we can access `this.responseText` to see the full
+body of the response from our XHR request.
+
+Now that we know how to access the response, let's do something with it.
+
+### Parsing the XHR Response
+
+Since the Github API deals strictly in JSON, we know that our response
+will be well-formed JSON, so it should be easy for us to work with.
+
+Let's parse this response and list out the repositories on the page.
+We'll start by giving ourselves a place in the DOM to put the data.
+
+```html
+<div>
+  <h3>Repositories</h3>
+  <a href="#" onclick="getRepositories()">Get Repositories</a>
+  <div id="repositories"></div>
+</div>
+```
+
+Then let's start by simply listing the repository names.
+
+```js
+function showRepositories(event, data) {
+  console.log(this.responseText)
+  var repoList = "<ul>"
+  for(var i=0;i < this.responseText.length; i++) {
```

Comment:
can't do it here because `responseText` doesn't `map` but can def do it in the later places after we parse JSON.